### PR TITLE
[Post] Flutter Web: A Fractal of Bad Design

### PR DESCRIPTION
**Pull request by k0nserv**

### CODE REVIEW

File: _posts/2020-10-31-flutter-web-a-fractal-of-bad-design.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title:  "Flutter Web: A Fractal of Bad Design"
+categories: cross-platform web html css opinion
+date: 2020-10-31
+description: >
+  How a misguided attempt to achieve cross platform UI leads to terrible experiences and kills the open web
+---
+
+The web has a long and rich history dating back to the nighties at CERN. Back then [Tim Berners-Lee](https://twitter.com/timberners_lee) laid the foundation of HTML that is still around today. There have been attempts to replace it with varying success but none have been successful, for good reason. HTML and the later invention of CSS are a remarkably powerful set of tools to build all kinds of experiences on the web. People are still trying to replace HTML, which brings us to the topic of this post: Flutter Web.
```
Comment (shaundon) [inline _posts/2020-10-31-flutter-web-a-fractal-of-bad-design.md]: nighties should be nineties

File: _posts/2020-10-31-flutter-web-a-fractal-of-bad-design.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+I use the browser extension [Vimium](https://vimium.github.io/) to navigate the web, it's an amazingly powerful tool that relies on, you guessed it, semantic HTML. Does it work on pages built with Flutter Web? Fuck no. It doesn't work because it tries to find things that are semantically clickable, like `button` or `a` elements, of which, as we have established, Flutter Web generates none. Vimium works on almost all websites I use because most developers, thankfully, don't just stick `onClick` handlers on `div`s. However, whatever crazy shit Flutter Web does, doesn't look clickable. Things being clickable that don't look like it to is a good proxy for poor accessibility. For example, screen readers can navigate by landmarks, such as headings, **links**, forms, and other semantic elements, does any of this work with Flutter Web? Fuck no.
```
Comment (shaundon) [inline _posts/2020-10-31-flutter-web-a-fractal-of-bad-design.md]: "Look like it to" might be a typo?

File: _posts/2020-10-31-flutter-web-a-fractal-of-bad-design.md (hunk 1 above)
Comment (k0nserv) [reply _posts/2020-10-31-flutter-web-a-fractal-of-bad-design.md]: lol 🙈

File: _posts/2020-10-31-flutter-web-a-fractal-of-bad-design.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+## Conclusion
+
+Flutter is a misguided attempt to achieve the impossible: quality cross platform experiences. Flutter Web in particular is fundamentally flawed and needs to be rebuilt from the ground up if it has any hopes of being viable tech that generates semantic, accessible, and modern web experiences. I have serious doubt that when Flutter Web leaves beta any of this will be addressed properly, unless the whole approach is reconsidered. If you see Flutter Web, turn around and run in the opposite direction.
+
+Developers, designers, and product people all love cross platform solutions because it saves them time and energy while achieving the "same" outcome as the costlier alternatives. Flutter Web nicely illustrates that the outcomes aren't the same, the visible part of a product is only one part of the puzzle.
```
Comment (k0nserv) [inline _posts/2020-10-31-flutter-web-a-fractal-of-bad-design.md:78]: @shaundon I added this extra section

**Pull request merged**